cmd/gardener-extension-provider-openstack/app/app.go

```diff
@@ -212,6 +215,18 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not add controllers to manager: %w", err)
 			}
 
+			if err := mgr.AddReadyzCheck("informer-sync", gardenerhealthz.NewCacheSyncHealthz(mgr.GetCache())); err != nil {
+				return fmt.Errorf("could not add readycheck for informers: %w", err)
+			}
+
+			if err := mgr.AddHealthzCheck("ping", healthz.Ping); err != nil {
```

Comment:
If I recall correctly this `healthz.Ping` returns always nil, which means the health check/liveness probe will always be true, right?

Reply:
Yes.